iphone/cli/commands/_build.js

```diff
@@ -6842,6 +6842,11 @@ iOSBuilder.prototype.invokeXcodeBuild = function invokeXcodeBuild(next) {
 		}
 	}
 
+	// Exclude arm64 architecture from simulator build in XCode 12+ - TIMOB-28042
+	if (this.target === 'simulator' && parseFloat(this.xcodeEnv.version) >= 12.0) {
+		args.push('EXCLUDED_ARCHS=arm64');
+	}
+	
```

Comment:
<!--
  1 failure:  iphone/cli/comman...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: iphone/cli/commands/_build.js;
  Line: 6849;
-->
- :no_entry_sign: iphone/cli/commands/_build.js line 6849 – Trailing spaces not allowed. (no-trailing-spaces)